### PR TITLE
test: add comprehensive tests for 5 core analyzers

### LINKED ISSUE #28: test: コアアナライザーのテスト追加
## 概要
5つのコアアナライザーの包括的なテストを追加し、テストカバレッジを80%以上に向上させる。

## 対象アナライザー
- `PerformanceAnalyzer` - 勝率、プロフィットファクター、シャープレシオ等
- `TaxAnalyzer` - Phantom Income（OID）、キャピタルゲイン、マルチ通貨
- `CostAnalyzer` - 手数料分析、コスト効率指標
- `RiskAnalyzer` - 金利シナリオ、集中リスク、VaR
- `BondAnalyzer` - YTM、デュレーション、STRIPS分析

## Acceptance Criteria
- [ ] `tests/analyzers/test_performance.py` を作成
- [ ] `tests/analyzers/test_tax.py` を作成
- [ ] `tests/analyzers/test_cost.py` を作成
- [ ] `tests/analyzers/test_risk.py` を作成
- [ ] `tests/analyzers/test_bond.py` を作成
- [ ] 各アナライザー: 正常系・異常系・境界値テストを含む
- [ ] Decimal精度テスト（float混入検出）
- [ ] アナライザー層のカバレッジ 80%以上を達成

## 技術要件
- `pytest` + `pytest-asyncio` 使用
- テスト用フィクスチャは `tests/fixtures/` に配置
- モックデータは実際のIBデータ形式に準拠

## 価値
機能追加・リファクタリング時の安全ネット。財務計算のバグを事前に検出。

## Wave
Wave 6（他のIssueと並列実行可能）

### PR DESCRIPTION
**Pull request by knishioka**
## Summary

- Add tests for `PerformanceAnalyzer`, `TaxAnalyzer`, `CostAnalyzer`, `RiskAnalyzer`, and `BondAnalyzer`
- 109 tests total across 5 new test files achieving **100% coverage** on all target analyzer modules
- Each file includes normal-case, boundary-value, error-case, and Decimal precision tests

Closes #28

## Details

| Test File | Tests | Coverage |
|-----------|-------|----------|
| `test_performance.py` | 20 | 100% |
| `test_tax.py` | 21 | 100% |
| `test_cost.py` | 23 | 100% |
| `test_risk.py` | 21 | 100% |
| `test_bond.py` | 24 | 100% |

### Test Categories Per Analyzer
- **Normal case**: Happy path with realistic IB data
- **Boundary values**: Empty accounts, single trade, all wins/losses, zero values, exact thresholds
- **Error case**: Missing account raises `ValueError`
- **Decimal precision**: All string values parse to `Decimal` without float contamination

### Key Patterns
- Follows existing `test_fx.py` conventions (fixtures, class-based organization)
- Uses `unittest.mock.patch` for `date.today()` in TaxAnalyzer and BondAnalyzer
- All financial values use `Decimal("...")` string literals

## Test plan

- [x] `ruff format --check` passes on all 5 files
- [x] `ruff check` passes with zero errors
- [x] `pytest tests/analyzers/` — 205 passed, 0 failed
- [x] Coverage: 100% on all 5 target analyzer modules
- [x] No scope creep: only test files added

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: tests/analyzers/test_tax.py [hunk 1]
```diff
@@ -0,0 +1,456 @@
...
+
+    def test_long_term_gains(self, mixed_tax_account):
+        analyzer = TaxAnalyzer(tax_rate=Decimal("0.30"), account=mixed_tax_account)
+        mock_date = _mock_date_today(date(2025, 6, 15))
+        with patch("ib_sec_mcp.analyzers.tax.date", mock_date):
+            result = analyzer.analyze()
+
+        # long_term_gain_trade: settle(2025-02-15) - trade(2024-01-10) = 401 days > 365
+        # and pnl=500 > 0 => long_term
+        assert Decimal(result["long_term_gains"]) == Decimal("500")
```
[the remark this answers is not in the document]
Comment (knishioka) [reply tests/analyzers/test_tax.py]: Fixed in commit d718630. Rewrote the test to document that it verifies current (known-inaccurate) behavior. Added a docstring explaining the limitation and a note referencing the TODO in `tax.py`. The assertion now uses `>= 0` instead of locking in a specific value, so a future holding-period fix won't appear as a regression.
